.travis.yml

```diff
@@ -12,6 +12,6 @@ install:
   - make deps
   - pip install coveralls
 script:
-  - make checks
+  - make events
```

Comment:
No change is required in `.travis.yml`, `Makefile`, and `README.rst` for this commit. This is just a `Makefile` target to run our coding style checks, unit tests, etc. In other words, `make checks`, runs the basic code quality checks before we push our commits and it is unrelated to the notion of *events* in the project.

I will provide the commands to remove changes to these three files from this commit in another comment.